src/apiflask/app.py

```diff
@@ -1193,3 +1201,20 @@ def _generate_spec(self) -> APISpec:
             spec.path(path=path, operations=sorted_operations)
 
         return spec
+
+    def _inject_decorators(self, config_name: str):
+        """inject the decorators in runtime.
+
+        Args:
+            config_name (str): inject config name
+        """
+        def wrapper(func):
+            @wraps(func)
+            def inject_decorators(*args, **kwargs):
+                spec_func = func
+                apispec_decorators = self.config.get(config_name, [])
```

Comment:
I would recommend using a general variable name here (i.e. `decorators`). The decorators could be fetched from three different configs (not just `APISPEC_DECORATORS`).

Reply:
Yes, it would be better.